dashbuilder-client/dashbuilder-widgets/src/test/java/org/dashbuilder/client/widgets/dataset/editor/DataSetDefPreviewTableTest.java

```diff
@@ -85,6 +85,29 @@ public void testDraw() throws Exception {
         verify(view, times(0)).init(tested);
         verify(view, times(0)).clear();
     }
-    
-    
+
+    @Test
+    public void testCSVConfig() throws Exception {
+        DataSetDef dataSetDef = DataSetDefFactory.newCSVDataSetDef()
+                .datePattern("dd/MM/yyyy")
+                .numberPattern("#,###")
+                .allColumns(true)
+                .buildDef();
+
+        DataSet dataSet = DataSetFactory.newDataSetBuilder()
+                .date("date")
+                .number("number")
+                .row(new Date(), 1d)
+                .buildDataSet();
+
+        when(dataSetLookupServices.lookupDataSet(any(), any())).thenReturn(dataSet);
+        tested.show(dataSetDef, null, displayerListener);
+
+        ArgumentCaptor<Displayer> argumentCaptor = ArgumentCaptor.forClass(Displayer.class);
+        verify(displayerListener).onDataLoaded(argumentCaptor.capture());
+        Displayer displayer = argumentCaptor.getValue();
+        DisplayerSettings settings = displayer.getDisplayerSettings();
```

Comment:
This particular commit looks like a fix to https://issues.jboss.org/browse/DASHBUILDE-202 not to DASHBUILDE-209. Could you please update the commit title (if these commits are not going to be squashed in the end)?